tox.ini

```diff
@@ -1,6 +1,6 @@
 
 [tox]
-envlist = py{27,34,35,36}-pytest{3,4}
+envlist = py{36,37,38,39}-pytest{3,4}
```

Comment:
Have you tried the newer versions of pytest?

Reply:
i ran the tests with pytest 6, i have added pytest versions 5 and 6 to the tox.ini, let me know if anything is off please

Reply:
Looks good!